src/modules/User.js

```diff
@@ -0,0 +1,26 @@
+import { DataTypes } from 'sequelize';
+import { client } from '../untils/db.js';
+
+export const User = client.define('users', {
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true,
+  },
+  password: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  activationToken: {
+    type: DataTypes.STRING,
+  },
+  userName: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  resetToken: {
+    type: DataTypes.STRING,
+    allowNull: true,
+    defaultValue: null,
```

Comment:
The `resetToken` field already has `allowNull: true`, so the `defaultValue: null` is redundant. You can remove `defaultValue: null` for clarity.